### PR TITLE
Added SassUtils for better compatibility with bootstrap 4 (and scss comments overall)

### LINKED ISSUE #779: Circular reference when assetic/sass compile bootstrap V4_alpha
Hi,

There is an error in assetic lib when compiling bootstrap V4 (with sass default ruby compiler, symfony on windows).

I think this comment in "bootstrap v4_alpha"  _reboot.scss line 47 causes a circular reference assetic compilation error :
// `.class-name { @import "bootstrap"; }`).

bootstrap.scss => _reboot.scss => bootstrap.scss => _reboot.scss => ...

Assetic library probably ignore comment and try to load bootstrap.scss. Error occur only in browser mode when reloading symfony web page.

"php app/console assetic:dump" command well produce the bootstrap.css ... strange

Stack trace :

```
...
[23-Nov-2015 00:44:55 UTC] PHP  10. Assetic\Factory\AssetFactory->getLastModified($asset = class Assetic\Asset\FileAsset { private $source = '../vendor/twitter/bootstrap/scss/_reboot.scss'; private ${Assetic\Asset\BaseAsset}:filters = class Assetic\Filter\FilterCollection { private $filters = array (...) }; private ${Assetic\Asset\BaseAsset}:sourceRoot = '../vendor/twitter/bootstrap/scss'; private ${Assetic\Asset\BaseAsset}:sourcePath = '_reboot.scss'; private ${Assetic\Asset\BaseAsset}:sourceDir = '../vendor/twitter/bootstrap/scss'; private ${Assetic\Asset\BaseAsset}:targetPath = '_controller/assetic/da4918c__reboot_1.scss'; private ${Assetic\Asset\BaseAsset}:content = NULL; private ${Assetic\Asset\BaseAsset}:loaded = FALSE; private ${Assetic\Asset\BaseAsset}:vars = array (); private ${Assetic\Asset\BaseAsset}:values = array () }) vendor\kriswallsmith\assetic\src\Assetic\Factory\AssetFactory.php:278

[23-Nov-2015 00:44:55 UTC] PHP  11. Assetic\Factory\AssetFactory->getLastModified($asset = class Assetic\Asset\FileAsset { private $source = '../vendor/twitter/bootstrap/scss/bootstrap.scss'; private ${Assetic\Asset\BaseAsset}:filters = class Assetic\Filter\FilterCollection { private $filters = array (...) }; private ${Assetic\Asset\BaseAsset}:sourceRoot = '../vendor/twitter/bootstrap/scss'; private ${Assetic\Asset\BaseAsset}:sourcePath = 'bootstrap.scss'; private ${Assetic\Asset\BaseAsset}:sourceDir = '../vendor/twitter/bootstrap/scss'; private ${Assetic\Asset\BaseAsset}:targetPath = '_controller/assetic/8f2cb7d_bootstrap_1.scss'; private ${Assetic\Asset\BaseAsset}:content = NULL; private ${Assetic\Asset\BaseAsset}:loaded = FALSE; private ${Assetic\Asset\BaseAsset}:vars = array (); private ${Assetic\Asset\BaseAsset}:values = array () }) vendor\kriswallsmith\assetic\src\Assetic\Factory\AssetFactory.php:278
```

This 2 lines repeat to the Max nested function call value.

### PR DESCRIPTION
**Pull request by andrewtch**
Fixes #779, #756.

### CODE REVIEW

**Comment by andrewtch** [comment]
Probably also closes #758 , #759.

@stof (or whoever is responsible for assetic in general), please note that I had to fix also Scssphp test since 5.3 and 5.6 versions of scssphp product different exceptions (is 5.3 really a must, it is EOL since October 2014, if I recall correctly?)

As I highly depend on fixing this bug and I have a bit of free time this week - feel free to point some other issues to PR before bumping a version )

File: src/Assetic/Util/SassUtils.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+namespace Assetic\Util;
+
+/**
+ * Sass Utils.
+ *
+ * @author Kris Wallsmith <kris.wallsmith@gmail.com>
+ */
+abstract class SassUtils extends CssUtils
+{
+    const REGEX_COMMENTS        = '/((?:\/\*[^*]*\*+(?:[^\/][^*]*\*+)*\/)|\/\/[^\n]+)/';
```
Comment (stof) [inline src/Assetic/Util/SassUtils.php]: please remove the extra spaces

**Comment by andrewtch** [comment]
@stof, fixed.

**Comment by kriswallsmith** [comment]
Thanks!

**Comment by andrewtch** [comment]
@kriswallsmith , any other issues or enhancements I can PR? As I said I have a bit of free time to work on OS projects.

**Pull request merged**